x-pack/plugins/security/public/management/api_keys/api_keys_grid/api_key_flyout.tsx

```diff
@@ -270,6 +271,12 @@ export const ApiKeyFlyout: FunctionComponent<ApiKeyFlyoutProps> = ({
                   ),
                 }}
                 fullWidth
+                onKeyDown={(e: React.KeyboardEvent<HTMLInputElement>) => {
+                  if (e.key === 'Enter' && formik.submitCount > 0) {
+                    e.preventDefault();
+                    e.stopPropagation();
+                  }
+                }}
```

Comment:
I'm not sure how I feel about this fix.

Reply:
If the form submission fails, will this allow a user to correct an error and resubmit the form?